test/integration/ocWrapper.test.ts

```diff
@@ -221,27 +231,35 @@ suite('./oc/ocWrapper.ts', function () {
 
         test('logout()', async function() {
             try {
-                await Oc.Instance.getCurrentUser();
-                expect.fail('should be unable to get current user, since you are logged out');
+                const needLogin = await LoginUtil.Instance.requireLogin();
+                expect(needLogin).to.be.true;
             } catch (_e) {
                 // do nothing
             }
         });
 
         test('loginWithUsernamePassword()', async function () {
-            await Oc.Instance.loginWithUsernamePassword(
-                clusterUrl,
-                username,
-                password,
-            );
-            const currentUser = await Oc.Instance.getCurrentUser();
-            expect(currentUser).to.equal(username);
+            if (isOpenShift) {
+                await Oc.Instance.loginWithUsernamePassword(
+                    clusterUrl,
+                    username,
+                    password,
+                );
+                const currentUser = await getCurrentUser();
+                expect(currentUser.trim()).to.equal(username);
+            } else {
+                this.skip
```

Comment:
I think you need to invoke `skip` as a method in order for it to work

Reply:
Yes, you're right